### ISSUE TITLE
bundleName_ should legalize

### ISSUE THREAD

**Issue by navidR** [pytorch/glow#3725]
When you don't legalize the bundleName_ in `LLVMIRGen` class then if passed name was wrong then the generated function would not be a valid C/C++ function name.

Look at this example:

```
#ifdef __cplusplus
extern "C" {
#endif

// Bundle memory configuration (memory layout)
extern BundleConfig 1conv2d_model.onnx_config;

// Bundle entry point (inference function)
void 1conv2d_model.onnx(uint8_t *constantWeight, uint8_t *mutableWeight, uint8_t *activations);

#ifdef __cplusplus
}
#endif
#endif

```

I will be sending a small patch to fix this error in a few hours.

**Comment by navidR** [comment]
https://github.com/pytorch/glow/pull/3726

**Comment by mciprian13** [comment]
Solved in #3901.
